jetty-ee9/jetty-ee9-nested/src/main/java/org/eclipse/jetty/ee9/nested/ContextHandler.java

```diff
@@ -1000,17 +1000,17 @@ protected void requestDestroyed(Request baseRequest, HttpServletRequest request)
         if (!_servletRequestListeners.isEmpty())
         {
             final ServletRequestEvent sre = new ServletRequestEvent(_apiContext, request);
-            for (int i = _servletRequestListeners.size(); i-- > 0; )
+            for (ServletRequestListener listener : TypeUtil.reverse(_servletRequestListeners))
```

Comment:
I'm a big -1 on this unrelated change.
On every request you could be copying and reversing several lists.

Reply:
Since the reverse iteration is being taken care of in #12297, that concern should probably be completely removed from this PR.

Reply:
@lorban this PR is already merged.